packages/ibm-products/src/components/TagSet/TagSet.tsx

```diff
@@ -499,6 +508,10 @@ TagSet.propTypes = {
    * display tags in multiple lines
    */
   multiline: PropTypes.bool,
+  /**
+   * Changes the behavior that occurs on selecting the overflow tag component
```

Comment:
i think we need to change this to be a little more specific because i don't think it clearly describes what happens when you use this prop. maybe something like `an optional click handler that replaces the default display all tags modal functionality`

Reply:
Ive updated all the relevant descriptions accordingly. Thanks for the suggestion.